sagemaker-training-compiler/huggingface/tensorflow_tune_batch_size/finding_max_batch_size_for_model_training.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>Add a little more explanation about how you're finding the max batch size and what it means, how to use it in the future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Added more explanation.